Account/FxADevice.swift

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+public struct FxADevice {
```

Comment:
Mentioned above but it's a bit confusing having this named FxADevice as well as the response wrapper. Best to keep them named differently if we can.